scripts/fuzz_opt.py

```diff
@@ -1369,6 +1370,23 @@ def handle(self, wasm):
         second_wasm = abspath('second.wasm')
         run([in_bin('wasm-opt'), second_input, '-ttf', '-o', second_wasm] + GEN_ARGS + FEATURE_OPTS)
 
+        # the second wasm file must not have an export that can influence our
+        # execution. the JS exports have that behavior, as when "table-set" is
+        # called it will look for the export "table" on which to operate, then
+        # imagine we lack that export in the first module but add it in the
+        # second, then code that failed before will now use the exported table
+        # from the second module (and maybe work). to avoid that, remove the
+        # table export, if it exists (and if the first module doesn't export
+        # it).
+        second_exports = get_exports(second_wasm, ['func', 'table'])
```

Comment:
What is the purpose of `'func'` here?

Reply:
We use `second_exports` later down, to pick the exports we want (all but the table). So we need the list of all exports here.